tests/test_chiral_restraints.py

```diff
@@ -265,6 +265,9 @@ def test_chiral_restraints_tetrahedral():
     )
 
     ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
+    # pessimize: strengthen harmonic angle force constant to max
+    ff.ha_handle.params[:, 0] = np.max(ff.ha_handle.params[:, 0])
```

Comment:
>To address the [Maybe] above, investigate how strong a chiral restraint has to be to flip over a mis-initialized geometry.

This test currently addresses a similar question (but in isolation).

(Just pessimized it a little in terms of HarmonicAngle force constant, but this test does **not** yet check that chiral_atom_k is strong enough to invert in context of bulkier molecule or more complicated environment.)